liveblog/liveblog
# [LBSD-2818] - Admin checkbox is missing and command adjustment

### Purpose

This PR allows for users with `is_support` to view the admin checkbox. This PR is accompanied by this [PR](https://github.com/liveblog/superdesk-core/pull/17) which makes the command adjustment.

### What has changed
- Added check to show the admin checkbox form to user with `is_support` set to true 

### Steps to test
1. Create a new user using the `--support` dash to enable support i.e
```
python3 manage.py users:create -u adminTwo -p adminTwo -fn Admin -ln Admin -e 'adminTwo@example.com' --support;
```
2. Log into the Editor with the username and password.
3. Navigate to User Management.
4. Click on any user or create user. Observe that you can view the Admin checkbox.


Resolves: [LBSD-2818](https://sofab.atlassian.net/browse/LBSD-2818)

[LBSD-2818]: https://sofab.atlassian.net/browse/LBSD-2818?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
